prime-router/src/main/kotlin/Report.kt

```diff
@@ -259,6 +265,10 @@ class Report {
         return StringColumn.create(name, List(itemCount) { "" })
     }
 
+    fun getBodyFormat(): OrganizationService.Format {
+        return this.destination?.format ?: OrganizationService.Format.CSV
+    }
+
```

Comment:
Again, this function was in a different place - it was convenient and made sense for a report to know its own format.